src/tests/ftest/telemetry/pool_space_metrics.py

```diff
@@ -0,0 +1,177 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from ior_test_base import IorTestBase
+from telemetry_test_base import TestWithTelemetry
+
+# Pretty print constant
+RESULT_OK = "[\033[32m OK\033[0m]"
+RESULT_NOK = "[\033[31mNOK\033[0m]"
+
+# It should take at much 10s for the vos space metrics to be up to date
```

Comment:
```suggestion
# It should take as much as 15s for the vos space metrics to be up to date
```
typo?

Reply:
Yes and no, the 10s is correct... however, my wonderful frenglish _at much_ is invalid.
I will fix and repush the PR as soon as the functional tests have been validated by the CI.

Reply:
- [ ] Fix typo